llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -5709,6 +5711,46 @@ SDValue AArch64TargetLowering::getRuntimePStateSM(SelectionDAG &DAG,
                      Mask);
 }
 
+static unsigned getIntrinsicID(const SDNode *N);
```

Comment:
Doesn't this already exist? It's used in other places in this file.

Reply:
It's not defined until a few K lines below this